scripts/start_local

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+PROJECT_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))/.."
+DOCKER_COMPOSE_FILE="${PROJECT_ROOT}/docker-compose.yml"
+DOCKER_COMPOSE_LOCAL_FILE="${PROJECT_ROOT}/docker-compose.local.yml"
+
+if ! type "mkcert" &> /dev/null; then
+  echo "Could not find mkcert, which is required for generating locally-trusted TLS certificates. Follow the installation instructions at https://github.com/FiloSottile/mkcert, then run this script again."
+  exit 1
+fi
+
+mkcert -install
+mkcert \
```

Comment:
should we check if certificate already exists before calling `mkcert`?

Reply:
It doesn't hurt anything to recreate it, and doing so will just extend the life of it anyway. (Not that it's got a short life.)